examples/other/integration_pytorch.py

```diff
@@ -0,0 +1,530 @@
+# run_example = false
+
+"""
+Pytorch Integration
+=============================
```

Comment:
The underline must match the heading to satisfy Sphinx.
```suggestion
Pytorch Integration
===================
```

Reply:
Thanks, will be addressed.